### PR TITLE
Fixes and optimises apc_control.dm a tad.

### LINKED ISSUE #37211: Power flow control console does not open APC TGUI interface
Revision: 2570384371bd9a92cedb1db979c52a0e13128193

Normally, the CE office power flow control console (/obj/machinery/computer/apc_control) lets you remotely open APC interfaces.

Now, the TGUI interface doesn't open at all. Everything else is fine - console messages, admin log, etc.

`Connected to APC in Chief Engineer's Office. Interface request sent.`
`ADMIN LOG: CKEY/(Jeb Ray) remotely accessed CE Office APC from the power flow control console at Chief Engineer's Office.`
`CE Office APC beeps, "Remote access detected. Interface unlocked."`
`Disconnected from CE Office APC.`

### PR DESCRIPTION
**Pull request by Jalleo**
it can probably be fixed up further.

Fixes #37211

:cl: Power Control Support
fix: What do you mean we accidentally routed the interaction back to the apc control conso- oh for fleeps sake STEVE WHAT DID YOU DO!!
/:cl:

I also removed a list inside the computer for this object type since it literally wasnt nessecary and getting it from the GLOB list when you interact with the computer is a much saner and more suitable route.

### CODE REVIEW

File: code/game/machinery/computer/apc_control.dm [hunk 1]
```diff
@@ -130,7 +124,7 @@
 			active_apc = null
 		to_chat(usr, "<span class='robot notice'>[icon2html(src, usr)] Connected to APC in [APC.area]. Interface request sent.</span>")
 		log_activity("remotely accessed APC in [APC.area]")
-		APC.interact(usr, GLOB.not_incapacitated_state)
+		APC.ui_interact(usr, state = GLOB.not_incapacitated_state)
```
Comment (Cyberboss) [inline code/game/machinery/computer/apc_control.dm]: user*

**Review by Cyberboss — APPROVED** [review]

**Review by Cyberboss — CHANGES_REQUESTED** [review]
^

File: code/game/machinery/computer/apc_control.dm (hunk 1 above)
Comment (Jalleo) [reply code/game/machinery/computer/apc_control.dm]: Gotcha will do that on Sunday

Comment (ShizCalev) [reply code/game/machinery/computer/apc_control.dm]: `usr` isn't used anywhere else in this topic()

Comment (Jalleo) [reply code/game/machinery/computer/apc_control.dm]: True but I can probably convert it over.

Comment (Cyberboss) [reply code/game/machinery/computer/apc_control.dm]: no this is my bad, i didn't realize the proc changed

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**